x/blob/README.md

```diff
@@ -64,8 +64,7 @@ value is set below that of normal transaction gas consumption, which is 10.
 #### `GovMaxSquareSize`
 
 `GovMaxSquareSize` is the maximum size of a data square that is considered valid
-by the validator set. This value is superseded by the `MaxSquareSize`, which is
-hardcoded and cannot change without hardforking the chain. See
+by the validator set. See
```

Comment:
I think we should still mention the upper bound but instead say something like:
"This value is capped by the `SquareSizeUpperBound`, which is a versioned constant."

Reply:
I think that phrasing is misleading because `GovMaxSquareSize` isn't actually capped by `SquareSizeUpperBound`. See https://github.com/celestiaorg/celestia-app/issues/2185

Reply:
True but I think we should somehow mention that the effective size of the square can't be exceeded by that value even if you change `GovMaxSquareSize` beyond it

Reply:
Makes sense. Evan had this idea: https://github.com/celestiaorg/celestia-app/issues/2267